webpack.common.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     background: path.resolve('src/background/background.ts'),
     contentScript: path.resolve('src/contentScript/contentScript.ts'),
     utils: path.resolve('src/utils/utils.ts'),
-    utils: path.resolve('src/utils/queries.ts'),
+    utils: path.resolve('src/utils/storageClasses.ts'),
```

Comment:
you don't need to add every file you have to webpack only the content/background or any other script you have, since they import these files webapp automatically bundles them together